nautobot/extras/tests/test_views.py

```diff
@@ -568,6 +569,25 @@ def setUpTestData(cls):
         }
 
 
+class ExternalIntegrationTestCase(ViewTestCases.PrimaryObjectViewTestCase):
```

Comment:
It would be nice to have some test cases for the bad format of the `remote_url` field and < 0 time for the `timeout` field. But not necessary